frontend/scripts/robot/actions/edit.js

```diff
@@ -28,12 +28,16 @@ export default function edit(model) {
       if (response instanceof Error) {
         throw response;
       } else {
-        let status = response.statusText;
+        let loadError = {
```

Comment:
I decided to live `oadError` object and I put it to `Error` component to update DocumentTitle